docs/tutorials/transformer.ipynb

```diff
@@ -710,7 +710,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Note: Take note that this mask indicates which entries to *not* attend to. The newly added MultiHeadAttention layer in Keras considers the invered mask, where 1 is a token to be attended to and 0 should be ignored. If you are using `tf.keras.layers.MultiHeadAttention` instead of the MultiHeadAttention layer in this tutorial defined below, use the following commented out method to generate the mask. You will need to `pip install tensorflow-probability` and `import tensorflow_probability as tfp`."
+    "Note: This mask indicates which entries to *not* attend to. However, the MultiHeadAttention layer in Keras that was added after the creation of this tutorial considers the invered mask, where 1 is a token to be attended to and 0 should be ignored. If you are using `tf.keras.layers.MultiHeadAttention` instead of the MultiHeadAttention layer in this tutorial defined below, use the following commented out method to generate the mask. You will need to `pip install tensorflow-probability` and `import tensorflow_probability as tfp`."
```

Comment:
Looks good! can you just fix the spelling of inverted in "...considers the invered mask,"

Reply:
Change was made, sorry for the delay